examples/todos/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "MIT",
   "dependencies": {
     "@elastic/synthetics": "*",
+    "playwright-chromium": "^1.6.2",
```

Comment:
This would straight away blow up and would not work. An easy way to test the breakage would be to change the

```
"@elastic/synthetics": "0.5.0" //does not depend on 1.6.2 chromium
"playwright-chromium": "^1.6.2"
```

The reason why this would break is because of the installation of different versions of chromium browser on global installation path. 

We would need to assume the image has already preinstalled node_modules directory with specified playwright version from installing `@elastic/synthetics` and remove `playwright-chromium` from deps. You could also do force install of `@elastic/synthetics` if you want to be absolutely sure about installing the right version and triggering pre install hook from playwright.